fury/shaders/billboard_dec.frag

```diff
@@ -1,3 +1,2 @@
 /* Billboard  fragment shader declaration */
-in vec3 centeredVertexMC;
-in float scalingFactor;
+in vec3 normVertexMCVSOutput;
```

Comment:
Not sure that `norm` is a good name, We can easily confuse it with normal. I personally preferred `centered`

Reply:
Will `normalized` be a better choice? I think `centered` is not an accurate description of what that variable is supposed to be.

Reply:
ok, it seems we have no choice to put the whole word (`normalized`)